open_dread_rando/door_patcher.py

```diff
@@ -117,17 +117,21 @@ class DoorType(Enum):
     WAVE_BEAM = ("wave_beam", ActorData.DOOR_POWER, True, ActorData.SHIELD_WAVE_BEAM)
     MISSILE = ("missile", ActorData.DOOR_POWER, True, ActorData.SHIELD_MISSILE)
     SUPER_MISSILE = ("super_missile", ActorData.DOOR_POWER, True, ActorData.SHIELD_SUPER_MISSILE)
-    GRAPPLE = ("grapple_beam", ActorData.DOOR_GRAPPLE)
-    PRESENCE = ("phantom_cloak", ActorData.DOOR_PRESENCE, False, None, True, False)
+    GRAPPLE = ("grapple_beam", ActorData.DOOR_GRAPPLE, False, None, True, True, ["actors/props/door"])
+    PRESENCE = ("phantom_cloak", ActorData.DOOR_PRESENCE, False, None, True, False, ["actors/props/door"])
 
-    def __init__(self, rdv_door_type: str, shield_data: ActorData, need_shield: bool = False,
-                 shield_actordef: ActorData = None, can_be_removed: bool = True, can_be_added: bool = True):
+    def __init__(self, rdv_door_type: str, door_data: ActorData, need_shield: bool = False,
+                 shield_data: ActorData = None, can_be_removed: bool = True, can_be_added: bool = True, additional_asset_folders: list[str] = []):
```

Comment:
Don't use `[]` as a default argument, especially not if you're going to modify it. It ends up being the same object in all calls, so weird things happens.